board_types.txt

```diff
@@ -49,6 +49,7 @@ TARGET_HW_PDW_MAS_MAIN-V1              86
 TARGET_HW_ATL_MANTIS_EDU               97
 TARGET_HW_THE_PEACH_K1                212
 TARGET_HW_THE_PEACH_R1                213
+TARGET_HW_ZEROONE_X6                  101
```

Comment:
remove this - these are for F4-F7 bootloaders H7 are built in tree

Reply:
Ok,I have removed this.